packages/components/src/duotone-picker/custom-duotone-bar.tsx

```diff
@@ -11,17 +11,25 @@ import {
 
 const PLACEHOLDER_VALUES = [ '#333', '#CCC' ];
 
-export default function CustomDuotoneBar( { value, onChange } ) {
+export default function CustomDuotoneBar( {
+	value,
+	onChange,
+}: {
+	value?: string[];
+	onChange: ( value?: string[] ) => void;
+} ) {
 	const hasGradient = !! value;
 	const values = hasGradient ? value : PLACEHOLDER_VALUES;
 	const background = getGradientFromCSSColors( values );
 	const controlPoints = getColorStopsFromColors( values );
 	return (
+		// @ts-expect-error Resolve after CustomGradientBar is migrated to TypeScript
 		<CustomGradientBar
 			disableInserter
 			background={ background }
 			hasGradient={ hasGradient }
 			value={ controlPoints }
+			// @ts-expect-error Resolve after CustomGradientBar is migrated to TypeScript
```

Comment:
@chad1008 another thing to add to #48929 's list

Reply:
We should be able to remove this comment (and the one a few lines above) since https://github.com/WordPress/gutenberg/pull/48929 was merged. Hopefully we don't need to make too many tweaks 🤞